pkg/api/latest/latest.go

```diff
@@ -28,17 +28,25 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// Version is the string that represents the current external default version.
+// GroupVersion is the string that represents the current external default groupVersion.
+var GroupVersion string
+
+//
+var Group string
+
+//Version is the string that represents the current external default Version.
 var Version string
 
 // OldestVersion is the string that represents the oldest server version supported,
 // for client code that wants to hardcode the lowest common denominator.
 var OldestVersion string
 
-// Versions is the list of versions that are recognized in code. The order provided
-// may be assumed to be least feature rich to most feature rich, and clients may
-// choose to prefer the latter items in the list over the former items when presented
-// with a set of versions to choose.
+// GroupVersions is the list of groupVersions that are recognized in code. The
+// order provided may be assumed to be least feature rich to most feature rich,
+// and clients may choose to prefer the latter items in the list over the former
+// items when presented with a set of versions to choose.
+var GroupVersions []string
```

Comment:
GroupVersions only contains groupVersions for one group (it's for the monolithic API in this case). I will change the comment to make this clear.